androidClient/app/src/main/java/com/macbitsgoa/comrades/coursematerial/ItemCourseMaterial.java

```diff
@@ -0,0 +1,66 @@
+package com.macbitsgoa.comrades.coursematerial;
+
+import java.util.ArrayList;
+
+/**
+ * @author Aayush singla
+ */
+
+public class ItemCourseMaterial {
+    private String addedBy;
```

Comment:
make all fields public, remove getters and setters

Reply:
this is a better way of doing that

Reply:
No, it's of no use having private fields and public getters and setters for all of them. It's just more redundant code.

Reply:
No, it's of no use having private fields and public getters and setters for all of them. It's just more redundant code.